src/Raven.Client/Constants.cs

```diff
@@ -136,6 +136,8 @@ private Metadata()
 
                 public const string Attachments = "@attachments";
 
+                public const string LegacyAttachmentsMetadata = "@legacy-attachment-metadata";
```

Comment:
aren't we deleting `@` properties on save?

Reply:
Do we? if so its not working since i saw this on the studio because i actually test my code before i send a PR

Reply:
If this is working, great